src/Components/WebAssembly/WebAssembly.Authentication/src/Interop/AuthenticationService.ts

```diff
@@ -113,15 +113,20 @@ class OidcAuthorizeService implements AuthorizeService {
 
                 const newUser = await this._userManager.signinSilent(parameters);
 
+                if (hasValidAccessToken(newUser) && hasAllScopes(request, newUser.scopes)) {
```

Comment:
I'm not sure about this check, I think it can probably be simplified to `newUser && newUser.access_token` and not worry about the scopes.